RustFxA/FxAWebViewController.swift

```diff
@@ -144,7 +130,7 @@ extension FxAWebViewController: WKUIDelegate {
         helpBrowser = wv
         helpBrowser?.navigationDelegate = self
 
-        navigationItem.leftBarButtonItem = UIBarButtonItem(title: FxAWebViewModel.BackTitle, style: .plain, target: self, action: #selector(closeHelpBrowser))
+        navigationItem.leftBarButtonItem = UIBarButtonItem(title: Strings.BackTitle, style: .plain, target: self, action: #selector(closeHelpBrowser))
```

Comment:
nit: capitalization was wrong on FxAWebViewModel.BackTitle, but also the helpviewer logic is already here, no reason to jump there for this.